src/components/StudentFeedback.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <section class="feedback-container">
+    <div class="feedback-container__category">
+      <p class="feedback-container__category--label">{{this.categories[0]}} </p>
+      <div class="progress-bar" id="bar1">3.5</div>
+      <p class="feedback-container__category--comments">
+        Solid work here! Blah blah.
+      </p>
+    </div>
+    <div class="feedback-container__category">
+      <p class="feedback-container__category--label">{{this.categories[1]}} </p>
+      <div class="progress-bar" id="bar2">2.5</div>
+      <p class="feedback-container__category--comments">
+        Some comments here about this category.
+      </p>
+    </div>
+    <div class="feedback-container__category">
+      <p class="feedback-container__category--label">{{this.categories[2]}} </p>
+      <div class="progress-bar" id="bar3">3</div>
+      <p class="feedback-container__category--comments">
+        Looking good. Just make sure to foo!
+      </p>
+    </div>
+    <hr class="feedback-container__dividing-line">
+    <div class="feedback-container__category">
+      <p class="
+        feedback-container__category--label
+        feedback-container__category--label-overall
+      ">
+        Overall
+      </p>
+      <div class="progress-bar" id="bar-average">3</div>
+      <p class="
+        feedback-container__category--comments
+        feedback-container__category--comments-overall
+      ">
+        You like cats because they are fat and fluffy inspect anything brought into the house, yet flee in terror at cucumber discovered on floor ask for petting trip on catnip. Run off table persian cat jump eat fish. Loves cheeseburgers meowing chowing and wowing and human is behind a closed door, emergency! abandoned! meeooowwww!!!.
+      </p>
+    </div>
+  </section>
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+      categories: ['JavaScript', 'React', 'Professionalism']
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+  @import '@/styles/styles.scss';
```

Comment:
Does this need to go here if it is also being imported in main.js?

Reply:
I don't think so. You have access to it through the main.js import.

Reply:
And a solid start! I wonder if we can make those category renders more dynamic? Categories is an array, so we should be able to use a v-for directive to iterate through them. Maybe we can make that an array of objects like the way they'll come in from the api, with a category, a score, and the category feedback (which I love, btw!!).

Reply:
I ended up needing to import here as well, so it looks like the main.js file import doesn't carry over into the components.

Reply:
Yes, let's absolutely make this more dynamic!!